internal/poker/games.go

```diff
@@ -0,0 +1,67 @@
+package poker
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"fmt"
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api/v5"
+)
+
+type (
+	gameHash string
+
+	games map[gameHash]game
+	game  struct {
+		votes     votes
+		createdAt int
+		updatedAt int
+		creator   *tgbotapi.User
+		jiraIssue string
```

Comment:
I do not like that it's "jira" we could but any string here or any issue tracker in the future.

Reply:
Sure, we can call it differently